Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,39 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  download "https://github.com/libfuse/libfuse"
+
+  satisfy(build_env: false) do
+    next true if libfuse_formula_exists? && Formula["libfuse"].installed?
+    includedirs = %w[
+      /usr/include
+      /usr/local/include
+    ]
+    next true if (includedirs.map do |dir|
+      File.exist?(File.join(dir, "fuse.h"))
+    end).any?
```

Comment:
Admittedly, I couldn't find a good way to write this that kept it below 120 chars. Sorry if this is kind of ugly, feel free to suggest something better or point me to the styleguide (I couldn't find it for the life of me...)